### ISSUE TITLE
Error building on openSUSE Leap 15.0

### ISSUE THREAD

**Issue by dradtke** [tinygo-org/tinygo#232]
I was trying to install tinygo on openSUSE Leap 15.0, which required adding the `devel:tools:compiler` repository to gain access to LLVM 7 and LLD, but now I'm getting a very cryptic error:

```
# runtime/cgo
/usr/bin/ld: /usr/bin/../lib64/crti.o: unable to initialize decompress status for section .debug_aranges
/usr/bin/ld: /usr/bin/../lib64/crti.o: unable to initialize decompress status for section .debug_aranges
/usr/bin/../lib64/crti.o: file not recognized: file format not recognized
clang-7.0.1: error: linker command failed with exit code 1 (use -v to see invocation)
```

Any thoughts on what the fix for this might be?

**Comment by dradtke** [comment]
After doing some more debugging, it looks like it might just be an issue with cgo, and only when running openSUSE inside Virtual Box (via Vagrant). If I can confirm that then I'll come back and close this issue.

**Comment by dradtke** [comment]
So it looks like it wasn't even an issue with being in a VM, since I destroyed it, created a new one, and managed to get it working. 🤷‍♂️ 

For posterity, the steps I followed to get tinygo to build on openSUSE Leap 15.0 are:

1. `sudo zypper ar obs://devel:tools:compiler compiler` to add the compiler tools repository
2. Install `llvm7-devel` and `clang7`
3. Set the C++ compiler to clang by doing `export CXX=clang`
4. Run `go get -u github.com/tinygo-org/tinygo`; this will fail initially
5. cd to the package source for `tinygo.org/x/go-llvm`
6. Modify the package's Makefile so that it runs `llvm-config` instead of `llvm-config-7`, and add `-lstdc++` to `LDFLAGS`
7. Run `make config` followed by `go build`
8. Reinstall tinygo

I haven't tested the command itself yet, but I've confirmed that it exists, and `tinygo help` displays the expected output.
